samples/dnn/dnn_model_runner/dnn_conversion/paddlepaddle/paddle_resnet50.py

```diff
@@ -0,0 +1,61 @@
+import paddle
+import paddlehub as hub
+import paddlehub.vision.transforms as T
+import cv2
```

Comment:
please, replace it with `import cv2 as cv` and then replace `cv2.` below with `cv.`

Reply:
Done